include/ir_cache/IRModuleCache.h

```diff
@@ -36,6 +36,10 @@ class IRModuleCache
                                  const std::string& func,
                                  const std::string& path);
 
+    size_t getSharedModuleDataSize(const std::string& user,
+                                   const std::string& func,
+                                   const std::string& path);
```

Comment:
This is here to give a bit more detail on the memory layout of shared modules (which is now a lot more explicit in the dynamic module loading)